### ISSUE TITLE
[android] Assert in UsageStats::EnterBackground()

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#4603]
I just clicked "Debug Android" in Android Studio:

```
abort 0x00000077e7885994
__assert2 0x00000077e7885d68
settings::UsageStats::EnterBackground() settings.cpp:406
Framework::EnterBackground() framework.cpp:1131
Java_app_organicmaps_MwmApplication_nativeOnTransit(JNIEnv *, jclass, jboolean) MwmApplication.cpp:59
art_quick_generic_jni_trampoline 0x000000753361a358
nterp_helper 0x000000753360939c
```

```
  // Safe check if uninitialized. Not sure, but possible if foreground/background calls
  // are not sequential during activity initialization.
  ASSERT(m_enterForegroundTime > 0, ());
  if (m_enterForegroundTime == 0)
    return;
```

3c1b2d1b2c5533be11a6d3e93efe42508f233c5d

**Comment by biodranik** [comment]
@rtsisyk it is called on moving the app to background. How is that possible after pressing "Debug" in Android Studio?

**Comment by matkoniecz** [comment]
is it fixed by #4615 by any chance?
